src/patternfly/components/Progress/progress.scss

```diff
@@ -200,10 +200,11 @@
 // It is assumed that the width of the progress__indicator is set inline or via Javascript
 .#{$progress}__indicator {
   position: absolute; // position absolute to bring above the semi-transparent bar overlay
- inset-block-start: 0;
- inset-inline-start: 0;
+  inset-block-start: 0;
+  inset-inline-start: 0;
```

Comment:
If this doesn't need to flip, we can just use `top: 0; left: 0;` and we won't need to add `direction`. Or you could just use `inset: 0;` since the inline `width` will limit the width. That's probably preferred because I think the linter will currently complain if you use a non-logical property.